newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/index.js

```diff
@@ -446,17 +451,27 @@ export default class ExpressionField extends React.Component<Props, State> {
                       true /* Can't use portals as this would put the Popper outside of the Modal, which is keeping the focus in the modal (so the search bar and keyboard browsing won't not work) */
                     }
                   >
-                    <Paper style={styles.expressionSelectorPopoverContent}>
-                      <ExpressionSelector
-                        selectedType=""
-                        onChoose={(type, expression) => {
-                          this._handleExpressionChosen(expression);
-                        }}
-                        expressionType={expressionType}
-                        focusOnMount
-                        scope={scope}
-                      />
-                    </Paper>
+                    <ResponsiveWindowMeasurer>
+                      {screenSize => (
+                        <Paper
+                          style={() => {
+                            screenSize === 'small'
+                              ? styles.expressionSelectorPopoverContentSmall
+                              : styles.expressionSelectorPopoverContent;
+                          }}
```

Comment:
This probably don't work because you passed a function, and it has no return.
You don't need the function here.